packages/shared/src/components/profile/Header.tsx

```diff
@@ -97,6 +100,27 @@ export function Header({
         />
         {!isSameUser && (
           <CustomFeedOptionsMenu
+            onAdd={(feedId) =>
```

Comment:
Wouldn't it be better to just do this inside the modal?
You don't need it anywhere else right?

Reply:
And then just pass entityId/entityType or something?
But

Reply:
We could. Personally I like making components as dumb as possible, especially if they are being reused. Easier to change stuff slightly if we deem necessary. 

If you guys think its better to just pass the props to the modal and do all the logic inside instead, I can make that quick change

Reply:
I am fine with leaving as is

Reply:
Cool let's leave it :)